pkg/apis/recognizer/devfile_recognizer_test.go

```diff
@@ -791,3 +857,31 @@ func getTestProjectPath(folder string) string {
 	basepath := filepath.Dir(b)
 	return filepath.Join(basepath, "..", "..", "..", "resources/projects", folder)
 }
+
+func TestCloseHttpResponseBody(t *testing.T){
+	tests := []struct {
+		name                string
+		url					string
+	}{
+		{
+			name:   "Closing File",
+			url: "http://www.google.com",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			resp, err := http.Get(tt.url)
+			if err != nil {
+				t.Errorf("Failed to get url")
+			}else {
+				closeHttpResponseBody(resp)
```

Comment:
Good catch, I think that's a good idea. Initially I implemented it this way because I didn't know to to do the http test properly but we got it now. I'll make that change